### PR TITLE
Downgrade dependency node-fetch to v2

### PR DESCRIPTION
**Pull request by OIRNOIR**

### CODE REVIEW

**Comment by OIRNOIR** [comment]
The package crashes any attempts at loading in CJS because it attempts to load version 3 or over of node-fetch, which is not compatible with CJS. Please do not update node-fetch to version 3 in the future to ensure compatibility. Version 2 is still in long-term support.

File: package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
 	"dependencies": {
 		"@sapphire/async-queue": "^1.5.0",
 		"ms": "^2.1.3",
-		"node-fetch": "^3.0.0"
+		"node-fetch": "^2.6.9"
```
Comment (thewilloftheshadow) [inline package.json]: Pin it please

**Review by thewilloftheshadow — APPROVED** [review]

**Review by thewilloftheshadow — CHANGES_REQUESTED** [review]
Pin the dependency

File: package.json (hunk 1 above)
Comment (OIRNOIR) [reply package.json]: Done

**Pull request merged**